middlewares/validators/task-requests.js

```diff
@@ -24,6 +27,7 @@ const postTaskRequests = async (req, res, next) => {
     .strict()
     .keys({
       externalIssueUrl: joi.string().regex(githubIssuerUrlPattern).required(),
+      externalIssueHtmlUrl: joi.string().regex(githubIssueHtmlUrlPattern).required(),
```

Comment:
required here and optional above, not getting it?

Reply:
fixed